modMcf/test/org/aion/mcf/account/KeystoreTest.java

```diff
@@ -195,4 +193,43 @@ public void testList() {
 
         assertTrue(hasAddr);
     }
+
+    @Test (expected = NullPointerException.class)
+    public void testBackupAccountWithNullInput(){
+        Keystore.backupAccount(null);
+    }
+
+    @Test (expected = NullPointerException.class)
+    public void testImportAccountNull(){
+        Keystore.importAccount(null);
+    }
+
+    @Test
+    public void testImportAccount(){
+        Map<String, String> importKey = new HashMap<>();
+        ECKey key = ECKeyFac.inst().create();
+
+        importKey.put(key.toString(), key.toString());
+        Set<String> res = Keystore.importAccount(importKey);
```

Comment:
Nothing gets asserted/verified here... did you forget to add them?

Also, right now when I run the test it doesn't go into the if-block at Keystore line 307 -- it just fails the if check and then the method exits.  I'm not sure if that is intentional but it seems to be skipping an important part of the code it is supposed to test.